fuzz/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "scrypt-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2018"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+pbkdf2 = { version = "=0.13.0-pre.1", path = "../pbkdf2" }
```

Comment:
I don't think you need all these dependencies. `libfuzzer-sys` and `scrypt` should be sufficient.